views/audit/README

```diff
@@ -0,0 +1,7 @@
+There are two views in this folder: query_audit and query_audit_v2
+query_audit is a view that simplifies querying audit logs to power dashboards, where the logs are version 1
+query_audit_v2 is also a view that simplifies querying audit logs, but operates for version 2 logs (BigQueryAuditMetadata)
```

Comment:
Version 1 and Version 2 are terms we use informally. 
Refer to them here by links to the documentation (e.g. BigQueryAuditMetadata section) and using old/new instead of v1/v2

Reply:
To clarify, keep the file names as is but clarify that query_audit_v2.sql refers to the BigQueryAuditMetadata (with link to docs)